editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -1450,10 +1452,8 @@ function parseElementProps(
 }
 
 type TSTextOrExpression = TS.JsxText | TS.JsxExpression
-type TSJSXElement = TS.JsxElement | TS.JsxSelfClosingElement
-type ElementsToParse = Array<
-  TSJSXElement | TSTextOrExpression | TS.JsxOpeningFragment | TS.JsxClosingFragment | TS.JsxFragment
->
+type TSJSXElement = TS.JsxElement | TS.JsxSelfClosingElement | TS.JsxFragment
```

Comment:
Interesting... Do we no longer need to cater for opening and closing fragments?

Reply:
Yes, originally we  treated the opening and closing fragments as separate tokens (siblings of their children), but now we handle the full fragment with its content as a level in the hierarchy.